tests/cli/workflow/nf_analysis/test_cli_config_case.py

```diff
@@ -32,25 +32,27 @@ def test_config_case_without_options(cli_runner: CliRunner, context: CGConfig, r
 
 
 @pytest.mark.parametrize(
-    "context",
-    ["rnafusion_context", "taxprofiler_context"],
+    "workflow",
+    [Workflow.RNAFUSION, Workflow.TAXPROFILER],
 )
 def test_config_case_with_missing_case(
     cli_runner: CliRunner,
     caplog: LogCaptureFixture,
     case_id_does_not_exist: str,
-    context: CGConfig,
+    workflow: Workflow,
     request,
 ):
     """Test config_case for Taxprofiler and Rnafusion with a missing case."""
```

Comment:
```suggestion
    """Test config_case for Workflow with a missing case."""
```

Reply:
In the spirit of adding future workflow we perhaps want this to be a general descriptor. Otherwise we might expand workflows here like rnafusion, and, and, etc.